src/ui/interaction/files/AvFileUpload/AvFileUpload.vue

```diff
@@ -0,0 +1,267 @@
+<script setup lang="ts">
+import type { Slot } from 'vue'
+import AvVIcon from '@/ui/base/AvVIcon/AvVIcon.vue'
+import { MDI_ICONS } from '@/ui/tokens'
+import { useRandomId } from '@gouvminint/vue-dsfr'
+
+/**
+ * AvFileUpload component props.
+ */
+export interface AvFileUploadProps {
+  /**
+   * Unique identifier for the file download component.
+   * If not specified, a random ID is generated.
+   *
+   * @default () => useRandomId(...)
+   */
+  id?: string
+
+  /**
+   * ARIA label for file download button.
+   *
+   * @default ''
+   */
+  ariaLabel?: string
+
+  /**
+   * Accepted file types, specified as a string (like HTML `accept` attribute)
+   * or an array of strings (which will be transformed into a string).
+   *
+   * @default undefined
+   */
+  accept?: string | string[]
+
+  /**
+   * Error message to be displayed in case of download problem.
+   *
+   * @default ''
+   */
+  error?: string
+
+  /**
+   * Message indicating that the downloaded file is valid.
+   *
+   * @default ''
+   */
+  validMessage?: string
+
+  /**
+   * Whether the file upload input is disabled.
+   *
+   * @default false
+   */
+  disabled?: boolean
+
+  /**
+   * Value linked to file upload input template.
+   *
+   * @default ''
+   */
+  modelValue?: string
```

Comment:
@p-beger un composant d'upload ne renvoie pas un string ça doit être File ou Buffer

Si j'ai bien compris tu v-model juste vers le nom du fichier.

et ton `change` event renvoie les fichiers 

```suggestion
  modelValue?: File[] | null
```

Reply:
https://vue-ds.fr/composants/DsfrFileUpload#%F0%9F%9B%A0%EF%B8%8F-props

<img width="700" height="126" alt="image" src="https://github.com/user-attachments/assets/91dfbe1a-7ec3-4f3a-9d3d-96a91d0a2578" />

Reply:
Est ce qu'on est obligé d'être ISO avec eux ?